components/modal/MatchManualModal.tsx

```diff
@@ -15,7 +24,7 @@ export default function MatchManualModal() {
       <div className={styles.textWrap}>
         🔍 매칭 <br />
         <div className={styles.textPadding}>
-          - 경기 시작 5분 전 상대 팀 공개 및 취소 불가 <br />
+          - 경기 시작 5분 전에 상대 팀 공개 및 취소 불가 <br />
```

Comment:
```suggestion
          - 경기 시작 5분 전부터 상대 팀 공개 및 취소 불가 <br />
```
'부터' 를 붙이는게 어떠할까요? :)

Reply:
매뉴얼은 전체적으로 수정할 때 다시 수정하겠습니다!